docs/tools.md

```diff
@@ -0,0 +1,738 @@
+- **Namespace Filtering**: Expose only specific namespaces using `--namespace` flags
+
+## Tool Categories
+
+### Management & Operations Tools
+- [Server Operations](#server-operations)
+- [Subscription Management](#subscription-management)
+- [Resource Group Operations](#resource-group-operations)
+- [Azure CLI Operations](#azure-cli-operations)
+- [Azure Developer CLI Operations](#azure-developer-cli-operations)
+- [Tools Discovery](#tools-discovery)
+
+### Data & Storage Tools
+- [Azure Storage Operations](#azure-storage-operations)
+- [Azure Cosmos DB Operations](#azure-cosmos-db-operations)
+- [Azure SQL Database Operations](#azure-sql-database-operations)
+- [Azure SQL Server Operations](#azure-sql-server-operations)
+- [Azure SQL Elastic Pool Operations](#azure-sql-elastic-pool-operations)
+- [Azure Database for PostgreSQL Operations](#azure-database-for-postgresql-operations)
+- [Azure Cache for Redis Operations](#azure-cache-for-redis-operations)
+
+### AI & Analytics Tools
+- [Azure AI Foundry Operations](#azure-ai-foundry-operations)
+- [Azure AI Search Operations](#azure-ai-search-operations)
+- [Azure Data Explorer Operations](#azure-data-explorer-operations)
+
+### Configuration & Security Tools
+- [Azure App Configuration Operations](#azure-app-configuration-operations)
+- [Azure Key Vault Operations](#azure-key-vault-operations)
+- [Azure RBAC Operations](#azure-rbac-operations)
+
+### Container & Orchestration Tools
+- [Azure Kubernetes Service (AKS) Operations](#azure-kubernetes-service-aks-operations)
+
+### Messaging & Integration Tools
+- [Azure Service Bus Operations](#azure-service-bus-operations)
+
+### Monitoring & Analytics Tools
+- [Azure Monitor Operations](#azure-monitor-operations)
+- [Azure Managed Grafana Operations](#azure-managed-grafana-operations)
+- [Azure Workbooks Operations](#azure-workbooks-operations)
+
+### Testing & Performance Tools
+- [Azure Load Testing Operations](#azure-load-testing-operations)
+
+### Marketplace & Third-Party Tools
+- [Azure Marketplace Operations](#azure-marketplace-operations)
+- [Azure Native ISV Operations](#azure-native-isv-operations)
+
+### Best Practices & Guidance Tools
+- [Azure MCP Best Practices](#azure-mcp-best-practices)
+- [Azure Terraform Best Practices](#azure-terraform-best-practices)
+
+---
+
+## Tool Details
+
+### Server Operations
+
+Core MCP server management and configuration tools.
+
+| Tool | Description | Read-Only | Parameters |
+|------|-------------|-----------|------------|
+| `azmcp-server-start` | Start the Azure MCP Server with specified configuration options | N/A | `transport`, `namespace[]`, `mode`, `read-only` |
+
+**Key Features:**
+- Multiple server modes (namespace, all, single)
+- Transport protocol selection (stdio, sse)
+- Namespace filtering capabilities
+- Read-only mode support
+
+---
+
+### Subscription Management
+
+Tools for managing Azure subscriptions and tenant access.
+
+| Tool | Description | Read-Only | Parameters |
```

Comment:
How about adding "Namespace" to the table so a user can lookup a tool and figure out which namespace it's in?